skyportal/utils/offset.py

```diff
@@ -1026,6 +1026,9 @@ def fits_image(
             ra=center_ra, dec=center_dec, imsize=imsize
         )
 
+    if url in [None, ""]:
```

Comment:
Should this error hard, or just add an entry in the log?

Reply:
it's ok to error here,  believe.